reference/shipping.v3.yml

```diff
@@ -355,6 +440,41 @@ components:
         - commodity_description
         - international_shipping
         - hs_codes
+    shippingMethods:
+      title: shippingMethods
+      description: Data about the customs information object.
+      type: object
+      x-internal: false
+      properties:
+        id:
+          description: The unique identifier of the shipping method.
+          type: integer
+          format: int32
+          example: 2
+        name:
+          description: The human-readable name of the shipping method.
+          type: string
+          example: "Courier delivery"
+        shipping_zone:
+          description: Zone information. This is available only when "include=shipping_zone" is present in the querystring.
+          type: object
+          properties:
+            id:
+              description: The unique identifier of the shipping zone.
+              type: integer
+              format: int32
+              example: 2
+            name:
+              description: The human-readable name of the shipping zone.
+              type: string
+              example: "Thailand"
+        channel_ids:
+          description: Channel information. This is available only when "include=shipping_zone" is present in the querystring.
```

Comment:
Would we not want to always include the Channel IDs? Well, perhaps only when the store has multiple channels?

Reply:
Yes probably a reasonable expecation. Updated!